scripts/logfetch/callbacks.py

```diff
@@ -14,6 +14,7 @@ def callback(response, **kwargs):
         sys.stderr.write(colored('Downloaded ', 'green') + colored(path, 'white') + '\n')
       else:
         sys.stderr.write(colored('.', 'green'))
+      f.close()
```

Comment:
pretty sure the `with` block should auto-close -- are you sure this is necessary?